portal/src/main/webapp/WEB-INF/jsp/cross_cancer_plots_tab.jsp

```diff
@@ -36,7 +36,7 @@
         width: 1160px;
     }
     #cc-plots-sidebar h5 {
-        margin-left: 20px;
+        margin-left: 10px;
```

Comment:
Change this to 5px and change `cc_plots_profile_list` width to 100px for better visualization.
Before:
<img width="1255" alt="screen shot 2017-05-25 at 12 06 39 pm" src="https://cloud.githubusercontent.com/assets/5400599/26458893/a5c6d09c-4142-11e7-904d-29b3267067ec.png">

After:
<img width="1247" alt="screen shot 2017-05-25 at 12 06 19 pm" src="https://cloud.githubusercontent.com/assets/5400599/26458890/a3140afe-4142-11e7-8ced-bc44757ec138.png">